Assets/GothicVR/Scripts/Items/SoundDaytime.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using GVR.World;
+using UnityEngine;
+
+namespace GothicVR.Items
+{
+    public class SoundDaytime : MonoBehaviour
+    {
+        
+#if UNITY_EDITOR
```

Comment:
Can you please remove this @JucanAndreiDaniel . It was a stupid idea on my side but I have feeling it's more questionable then helpful.

Reply:
Removed